lib/utils/EventEmitter/EventEmitter.js

```diff
@@ -0,0 +1,30 @@
+export class EventEmitter {
```

Comment:
Do we really need to use class? I understand idea of singleton, but in other hand your are using FP. But I don't know whole idea of it.

Reply:
discussed offline